web/src/app/modules/views/main/editors/modules/graphical-editor/components/graphical-editor.component.ts

```diff
@@ -79,7 +82,6 @@ export class GraphicalEditor {
         private undoService: UndoService,
         private modal: ConfirmationModal,
         private graphicalEditorService: GraphicalEditorService) {
-
         this.navigator.navigationStart.subscribe(() => {
             this.isInGraphTransition = true;
         });

```

Comment:
I am not that sure about the lifecycle in Angular, but should we unsubscribe the navigation events as well?

Reply:
I resolved this. See the last commit.